archon-ui-main/src/components/knowledge-base/CrawlingProgressCard.tsx

```
@@ -411,17 +420,25 @@ export const CrawlingProgressCard: React.FC<CrawlingProgressCardProps> = ({
           icon: <FileText className="w-4 h-4" />
         };
       case 'document_storage':
+        // Use stage-specific batch fields for document storage status display
+        const docCompleted = progressData.document_completed_batches ?? progressData.completedBatches;
+        const docTotal = progressData.document_total_batches ?? progressData.totalBatches;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)